cmd/deploy/deploy.go

```diff
@@ -316,34 +317,8 @@ func (dc *DeployCommand) RunDeploy(ctx context.Context, deployOptions *Options)
 		return err
 	}
 
-	// TODO: take this out to a new function deploy dependencies
-	for depName, dep := range deployOptions.Manifest.Dependencies {
-		oktetoLog.Information("Deploying dependency '%s'", depName)
-		dep.Variables = append(dep.Variables, model.EnvVar{
-			Name:  "OKTETO_ORIGIN",
-			Value: "okteto-deploy",
-		})
-		pipOpts := &pipelineCMD.DeployOptions{
-			Name:         depName,
-			Repository:   dep.Repository,
-			Branch:       dep.Branch,
-			File:         dep.ManifestPath,
-			Variables:    model.SerializeBuildArgs(dep.Variables),
-			Wait:         dep.Wait,
-			Timeout:      deployOptions.Timeout,
-			SkipIfExists: !deployOptions.Dependencies,
-		}
-		pc, err := pipelineCMD.NewCommand()
-		if err != nil {
-			return err
-		}
-		if err := pc.ExecuteDeployPipeline(ctx, pipOpts); err != nil {
-			if errStatus := updateConfigMapStatus(ctx, cfg, c, data, err); errStatus != nil {
-				return errStatus
-			}
-
-			return err
-		}
+	if err := dc.buildImages(ctx, cfg, c, data, deployOptions); err != nil {
+		return err
 	}
 
 	// TODO: take this out to a new function build images
```

Comment:
i'm see that most of the code from this block 325-344 has been moved to buildImages function, but this is still here ?

Reply:
Thanks for noticing that! In the separation of prs I forgot to delete those lines